dashboard/app/jobs/concerns/active_job_metrics.rb

```diff
@@ -35,49 +35,83 @@ def get_failed_job_count
     Delayed::Job.where.not(failed_at: nil).count
```

Comment:
Not directly related to this PR, but this file should probably be called `delayed_job_metrics.rb`, its not using any ActiveJob API ;-)